pipeline/build/build.go

```diff
@@ -49,7 +49,11 @@ func build(name, goos, goarch string, ctx *context.Context) error {
 			return err
 		}
 	}
-	cmd := []string{"go", "build", "-ldflags=" + ldflags, "-o", output, ctx.Config.Build.Main}
+	cmd := []string{"go", "build"}
+	if ctx.Config.Build.Flags != "" {
+		cmd = append(cmd, strings.Split(ctx.Config.Build.Flags, " ")...)
```

Comment:
Good call! Even though in this case it probably doesn't affect the behaviour much, it makes the intent more obvious :)